tests/Resources/ti.network.httpclient.addontest.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+const should = require('./utilities/assertions');
+
+describe('Titanium.Network.HTTPClient', function () {
+	this.timeout(6e4);
+
+	it.windowsMissing('.file set to a Ti.Filesystem.File object', function (finish) {
+		this.timeout(6e4);
+
+		const downloadedImageFile = Ti.Filesystem.getFile(Ti.Filesystem.applicationDataDirectory, 'DownloadedImage.png');
+		if (downloadedImageFile.exists()) {
+			downloadedImageFile.deleteFile();
+		}
+
+		const xhr = Ti.Network.createHTTPClient({});
+		xhr.setTimeout(6e4);
+		xhr.onload = function (e) {
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.network.httpclient.addontest.js;
  Line: 26;
-->

- :warning: tests/Resources/ti.network.httpclient.addontest.js line 26 – 'e' is defined but never used. Allowed unused args must match /^_.+/. (no-unused-vars)